frontend/src/components/Avatar/Avatar.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import { Box, MenuDivider, SimpleGrid } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+
+import DropdownMenu from '../Menu'
+
+import { Avatar } from './Avatar'
+
+export default {
+  title: 'Components/Avatar',
+  component: Avatar,
+} as Meta
+
+type AvatarGroupTemplateProps = {
+  fullName?: string
+  userName?: string
+  hasNotification?: boolean
+}
+
+type AvatarTemplateProps = {
+  fullName?: string
+  userName?: string
+  hasNotification?: boolean
+  isOpen?: boolean
+}
+
+const AvatarTemplate: Story<AvatarTemplateProps> = ({
```

Comment:
I don't think we need this for the Avatar story. The base avatar story should only be the Avatar component itself instead of all the bells and whistles

Reply:
edited